pkgs/applications/misc/double-commander/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  stdenv, lib
+  , fetchFromGitHub
+  , fpc, lazarus
+  , which, autoPatchelfHook
+  , libqt5pas, qt5, gtk2, dbus
+}:
+
+let qtLazarus = lazarus.override {
+  withQt = true;
+  inherit libqt5pas;
+};
+in
+stdenv.mkDerivation rec {
+  pname = "doublecmd";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "doublecmd";
+    repo = "doublecmd";
+    rev = "v${version}";
+    sha256 = "sha256-nX5EZekYF9saM0QaN9caCnsRDIqPYNX/BNu6Bb5lDYQ=";
+  };
+
+  nativeBuildInputs = [
+    fpc
+    which
+    autoPatchelfHook
+    qt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qt5.qtbase
+    libqt5pas
+    gtk2
+    dbus.lib
+  ];
```

Comment:
```suggestion
  buildInputs = [ libX11 glib dbus ]
    ++ lib.optionals (widgetset == "gtk2") [ gtk2 atk pango gdk-pixbuf cairo ]
    ++ lib.optional (widgetset == "qt5") libqt5pas;
```

Reply:
Thanks, applied in fe0fe8d6b0ebe88ed7bff6309c3b786592a9d2bd